lambda/BatchInferenceRunner.js

```diff
@@ -312,7 +312,9 @@ async function invokeTest(testId, user, batch, apiKey)
       {
         if (context.firstQueue !== undefined)
         {
-          if (context.firstQueue.payload === context.lastResponse.queue)
+          var regex = new RegExp(context.firstQueue.payload);
```

Comment:
This fixes the issue with queue tests failing due to string comparisons but regex written by default